### PR TITLE
Static demo viewer

### PR DESCRIPTION
**Pull request by will-moore**
Robot test of static viewer page to test bugs reported in
https://www.openmicroscopy.org/community/viewtopic.php?f=6&t=8058
cc @aleksandra-tarkowska 

To test manually:
- Go to https://eel.openmicroscopy.org/merge/webtest/static_example/
- Check that using the +, - and 1:1 buttons on each viewer ONLY zooms that viewer (and not both viewers) Fixed in https://github.com/openmicroscopy/openmicroscopy/pull/4701 
- Test that clicking thumbnails of 'Big' images then 'small' images loads them into viewport and viewport works for both. Fixed in https://github.com/openmicroscopy/openmicroscopy/pull/4695/

To run robot framework test:
- $ cd webtest
- Edit url in test/robot/test_static_viewer.txt to point to test server where webtest is installed, E.g. https://eel.openmicroscopy.org/merge/webtest/static_example/
- $ python -m robot.run test/robot/test_static_viewer.txt

### CODE REVIEW

File: test/robot/test_static_viewer.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+*** Settings ***
+
+Library           Selenium2Library
+
+
+
+*** Test Cases ***
+
+Test Static Viewer
+    Open Browser                        http://localhost:4080/webtest/static_example/       Chrome
```
Comment (atarkowska) [inline test/robot/test_static_viewer.txt:11]: @will-moore thanks for writing the test, would it be possible to replace the above with the standard setup like https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tests/ui/testcases/web/view_image.txt#L10

File: templates/webtest/demo_viewers/static_example.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
+<!doctype html>
+<!-- See discussion at https://www.openmicroscopy.org/community/viewtopic.php?f=6&t=8058&p=17114 -->
+<html>
+<head>
+
+    <link href="https://cowfish.openmicroscopy.org/webmerge/static/omeroweb.viewer.min.css" type="text/css" rel="stylesheet"></link>
```
Comment (atarkowska) [inline templates/webtest/demo_viewers/static_example.html]: I'd recommend to make a comment that is up to developer to come up with versioning strategy as that url will be cached in client web browser and may be outdated when OMERO.server is upgraded. Maybe we could add small JS script to put time-stamp in the end of min filed to make sure this is not cached by web browser

File: templates/webtest/demo_viewers/static_example.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    <script src="https://cowfish.openmicroscopy.org/webmerge/static/omeroweb.viewer.min.js" type="text/javascript"></script>
+
+    <!--
+    <script type="text/javascript" src="http://localhost:4080/static/3rdparty/jquery-1.11.1.js"></script>
+    <script type="text/javascript" src="http://localhost:4080/static/webgateway/js/ome.csrf.js?_5.3.0-m3-ice35-SNAPSHOT"></script>
```
Comment (atarkowska) [inline templates/webtest/demo_viewers/static_example.html]: The same here. If you want to include direct links I would suggest to use version number matching recent release and make a comment that is up to developer to maintain all these links

I am guessing we should also remove `localhost:4080` and replace it with something like `omero.host` (I am also interested what other people think about pointing to nightshade?)

**Comment by sbesson** [comment]
@will-moore: can you push the changes suggest by @aleksandra-tarkowska above. As in https://github.com/openmicroscopy/openmicroscopy/pull/4701#issuecomment-239803690, we can investigate the automated running of these web app tests as we grow them

**Comment by atarkowska** [comment]
This PR is conflicting with main line anyway, closing as #15 takes priority

**Pull request closed without merging**